plugins/git/git.plugin.zsh

```diff
@@ -32,6 +32,23 @@ function work_in_progress() {
     echo "WIP!!"
   fi
 }
+#change current branch old email with new eamil
+gmvemail() {
+  export OLD_GIT_EMAIL=$1
+  export NEW_GIT_EMAIL=$2
+  git filter-branch --commit-filter '
+    if [ "$GIT_AUTHOR_EMAIL" = "$OLD_GIT_EMAIL" ];
+    then
+            GIT_AUTHOR_EMAIL="$NEW_GIT_EMAIL";
+            git commit-tree "$@";
+    else
+            git commit-tree "$@";
+    fi' HEAD
+  unset OLD_GIT_EMAIL
```

Comment:
This isn't needed after you apply https://github.com/robbyrussell/oh-my-zsh/pull/5395/files#r78296768.